### ISSUE TITLE
TypeScript service dies in Quokka file in VSCode 1.66.0 

### ISSUE THREAD

**Issue by dhull** [wallabyjs/quokka#765]
### Issue description or question

Quokka seems to cause the TypeScript server to crash in VSCode 1.66.0

![Untitled-1 2022-03-31 11-39-57](https://user-images.githubusercontent.com/395287/161127131-5b9d9a7b-1ff5-49f3-b520-13af701973a7.png)

I created the file by hitting F1, selecting "Quokka.js: New file", and then "TypeScript".

Is this issue related to Quokka not outputting the expected results of your code?: No

### Sample code

None. The TypeScript server crashes even in an empty window.

### Code editor version

Visual Studio Code v1.66.0

### OS name and version

macOS 12.3

**Comment by smcenlly** [comment]
We cannot reproduce your issue.

Can you please try opening a new window in VS Code using the `New Window` command to see if you have the same issue when a project folder is not opened?

Can you also check if you have the same problem when you run the `File: New File` command and then select your editor language as `TypeScript` (this is what Quokka does when you select a new TypeScript file).

It looks like other people are experiencing the same problem outside of Quokka and it is related to VS Code Insiders and not Quokka: https://github.com/microsoft/vscode/issues/146325.

**Comment by dhull** [comment]
> Can you please try opening a new window in VS Code using the New Window command to see if you have the same issue when a project folder is not opened?

That's what I had done for my original report, but I didn't do as much investigation as I should have.

I tried disabling all of my extensions except for Quokka and the problem didn't happen. Then I tried re-enabling extensions and found that the problem is in Microsoft's TSLint extension (v1.3.3), and occurs even if TSLint is the only extension that is enabled. The problem seems to be between TSLint and the latest version of VSCode (1.66.0), which was just released.

Thank you for your help.
